sdks/python/setup.py

```diff
@@ -104,6 +104,7 @@ def get_version():
     'oauth2client>=2.0.1,<4.0.0',
     'protobuf>=3.2.0,<=3.3.0',
     'pyyaml>=3.12,<4.0.0',
+    'retrying==1.3.3',
```

Comment:
<!--new_thread; commit:eb2acfa855db99ab4b826344656ec29dcdb175e3; resolved:0-->
Can we avoid this dependency?  (see my other comments).